src/pykeen/triples/triples_factory.py

```diff
@@ -520,6 +521,18 @@ def _word_cloud(self, *, text: List[str], top: int):
         word_cloud = WordCloud()
         return HTML(word_cloud.get_embed_code(text=text, topn=top))
 
+    def tensor_to_df(self, tensor: torch.LongTensor) -> pd.DataFrame:
```

Comment:
will need an update to handle tensors that are not only 3 columns, as in the case when a score column (or other stuff, like novelties with respect to testing and training set) might be included